packages/scaffold-config/src/frameworks.ts

```diff
@@ -217,26 +217,27 @@ export const WIZARD_FRAMEWORKS = [
     supportStatus: 'full',
     componentIndexHtml: componentIndexHtmlGenerator(),
   },
-  {
-    type: 'angular',
-    configFramework: 'angular',
-    category: 'template',
-    name: 'Angular',
-    detectors: [dependencies.WIZARD_DEPENDENCY_ANGULAR_CLI],
-    supportedBundlers: [dependencies.WIZARD_DEPENDENCY_WEBPACK],
-    dependencies: (bundler: WizardBundler['type'], projectPath: string): DependencyToInstall[] => {
-      return [
-        inPkgJson(dependencies.WIZARD_DEPENDENCY_ANGULAR_CLI, projectPath),
-        inPkgJson(dependencies.WIZARD_DEPENDENCY_ANGULAR_DEVKIT_BUILD_ANGULAR, projectPath),
-        inPkgJson(dependencies.WIZARD_DEPENDENCY_ANGULAR_CORE, projectPath),
-        inPkgJson(dependencies.WIZARD_DEPENDENCY_ANGULAR_COMMON, projectPath),
-        inPkgJson(dependencies.WIZARD_DEPENDENCY_ANGULAR_PLATFORM_BROWSER_DYNAMIC, projectPath),
```

Comment:
i know this PR is reverting this code but I noticed that every single `inPkgJson` function is doing an `fs.readJsonSync` which seems unnecessary given you could just read the `package.json` once, assign it to a `const` and then traverse that object to pick out its values

Reply:
Created a follow up for addressing the sync calls in this package: https://github.com/cypress-io/cypress/issues/23016

Reply:
also these should not be sync calls because of the `ulimit` problem... needs to use graceful-fs aka regular `fs.readJson` since at this point we are already watching files and you could have crashes.